Intersect.Server/Entities/Events/Conditions.cs

```diff
@@ -457,8 +453,13 @@ QuestBase questBase
             QuestBase questBase
         )
         {
+            if (player == null)
```

Comment:
These are all implementations of `MeetsCondition`, which is a function that we call during event condition handling. They have to have the same signature because, by _this_ design, that's how we allow them to be kinda generic.

Reply:
Ahhh ok